lib/service/invoicesubscription.go

```diff
@@ -23,6 +23,32 @@ func (svc *LndhubService) ProcessInvoiceUpdate(ctx context.Context, rawInvoice *
 
 	svc.Logger.Infof("Invoice update: r_hash:%s state:%v", rHashStr, rawInvoice.State.String())
 
+	//Check if it's a keysend payment
+	//If it is, an invoice will be created on-the-fly
+	if rawInvoice.IsKeysend {
+		//First check if this keysend payment was already processed
```

Comment:
does it make sense to move this inner if into a special function?  - could maybe help with readability to make this whole method a bit shorter?